code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1990,9 +1990,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 		id = null
 
 /obj/item/device/pda/proc/create_message(var/mob/living/U = usr, var/obj/item/device/pda/P)
-
-
-	var/t = input(U, "Please enter message", name, null) as text
+	var/t = input(U, "Please enter message", "[P]", null) as text
```

Comment:
Excellent advice. This reminds that there's a bunch of other input()s that need a cancel button.